src/patternfly/demos/Table/table-collapsed-table.hbs

```diff
@@ -0,0 +1,228 @@
+{{#> table table--id=(concat page--id '-table') table--grid="true" table--modifier="pf-m-grid-xl" table--expandable="true" table--attribute='aria-label="Collapsed table example"'}}
+  {{#> table-thead}}
+    {{#> table-tr}}
+      {{#> table-td}}{{/table-td}}
+      {{#> table-th table-th--attribute='scope="col"' table-th--modifier="pf-m-width-30"}}
+        Repositories
+      {{/table-th}}
+      {{#> table-th table-th--attribute='scope="col"'}}
+        Branches
+      {{/table-th}}
+      {{#> table-th table-th--attribute='scope="col"'}}
+        Pull requests
+      {{/table-th}}
+      {{#> table-th table-th--attribute='scope="col"'}}
+        Work spaces
+      {{/table-th}}
+      {{#> table-th table-th--attribute='scope="col"'}}
+        Last commit
+      {{/table-th}}
+      {{#> table-td}}{{/table-td}}
+    {{/table-tr}}
+  {{/table-thead}}
+
+  {{#> table-tbody}}
+    {{#> table-tr}}
+      {{#> table-td table-td--toggle="true" table-td--button--attribute=(concat 'aria-labelledby="' table--id '-node1 expandable-toggle1" id="expandable-toggle1" aria-label="Details" aria-controls="' table--id '-content1"')}}{{/table-td}}
+      {{#> table-th table-th--data-label="Repository name"}}
+        <div>
+          <div id="{{table--id}}-node1">Node 2</div>
+          <a href="#">siemur/test-space</a>
+        </div>
+      {{/table-th}}
+      {{#> table-td table-td--data-label="Branches"}}
+        <span><i class="fas fa-code-branch"></i> 10</span>
```

Comment:
Might you use the flex layout here and for the other text + icons? 
```
<div class="pf-l-flex pf-m-space-items-sm">
  <i class="fas fa-code-branch"></i>
  <span>10</span>
</div>
```
**From this:**
![Screen Shot 2020-10-20 at 10 53 06 AM](https://user-images.githubusercontent.com/5385435/96604641-55a91880-12c3-11eb-94fe-be80df2901b0.png)

**To this:**
![Screen Shot 2020-10-20 at 10 54 40 AM](https://user-images.githubusercontent.com/5385435/96604706-678abb80-12c3-11eb-8b13-fbcd12ee1913.png)

Reply:
Actually, I think you could just add a `&nbsp;` and be fine

Reply:
@mattnolting wrapped the text in a span.